mappings/UofM.py

```diff
@@ -75,6 +70,40 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # Parse rights codes
+        rightsElements = self.record.rights.split('|') if self.record.rights else [''] * 5
```

Comment:
Hmm - one comment that might be helpful is an example of what the rights can look like and why there is this pipe delimiter.

Reply:
My recommendation is if self.record.rights is None then let's keep it as None otherwise we may have rights with empty strings.

Reply:
Ok I can leave a comment that does that.